env/Tensorflow_probability.lock

```diff
@@ -0,0 +1,313 @@
+# This file may be used to create an environment using:
+# $ conda create --name <env> --file <this file>
+# platform: linux-64
```

Comment:
lock files are platform specific essentiallym, so you may want to give this file a name like tensorflow_probaility_mo_linux.lock, so people know where they should expect it to work.

Reply:
Ah I didn't know this, I'll change the name